src/core/mount.c

```diff
@@ -1765,6 +1766,27 @@ static int mount_dispatch_io(sd_event_source *source, int fd, uint32_t revents,
         Iterator i;
         Unit *u;
         int r;
+        uint64_t this_run = now(CLOCK_MONOTONIC);
```

Comment:
as above: please use usec_t...

Also, according to our CODING_STYLE we don't like mixing variable declarations and function invocations in one line...